### ISSUE TITLE
Fails on boolean attributes

### ISSUE THREAD

**Issue by ForbesLindesay** [pugjs/babel-plugin-transform-react-pug#1]
e.g.

```
input(required)
```

**Comment by briancyzhang** [comment]
could you try the following?

`input(require='')`

**Comment by ForbesLindesay** [comment]
You can just do `input(required=true)` but we should support the shorthand.  We could alternatively make `input(value)` be a shorthand for `input(value=value)` though, and I'm not sure what the best option is.

**Comment by leviwheatcroft** [comment]
Additionally.. the error thrown for this is `Error: Unescaped attributes are not supported in react-pug` which didn't make any sense to me...

**Comment by ezhlobo** [comment]
@leviwheatcroft it's an interesting message. Could you provide your code example, so we can help you to handle it?

**Comment by leviwheatcroft** [comment]
Now I think about it, I was passing in react props rather than html attributes. I'm using `react-bootstrap` 

I didn't commit the code when it was producing that error but it was something like this:

```
return pug`
              Table(striped bordered condensed hover)
                thead
                  tr
                    th Created
                    th State
                tbody
                  each batch in batches
                    tr(key = batch.id)
                      td= batch.createdAtDisplay
                      td= batch.state
```

So `Table` is a react component from `react-bootstrap`

**Comment by ezhlobo** [comment]
@leviwheatcroft sorry for a long answer, I just missed it. The thing that now this plugin does not care about shorthands...

In your case you can do a workaround:

```pug
Table(striped=true bordered=true condensed=true hover=true)
```

BTW only one short attribute works fine:

```pug
Table(striped bordered=true condensed=true hover=true)
```

**Comment by leviwheatcroft** [comment]
Yeah that's the workaround I'm using, it's not a big deal and I can live without the shorthand syntax. the main thing I was bringing to your attention is that the error thrown doesn't really help you identify the problem. The error is: `Error: Unescaped attributes are not supported in react-pug`

**Comment by ezhlobo** [comment]
Would like to take a look at this issue closer after finishing with issue in pug-lexer (https://github.com/pugjs/pug/pull/2956).

I'm going to allow passing attributes as is (without shorthands). JSX will understand such attributes like boolean attributes with `true` value.

//CC @ForbesLindesay

**Comment by ezhlobo** [comment]
@ForbesLindesay I believe we can close this issue, since it's fixed in [v4.2](https://github.com/pugjs/babel-plugin-transform-react-pug/releases/tag/4.2.0).
